features/frameworks/TARGET_PSA/mbed_lib.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "psa-compliance",
+    "config": {
```

Comment:
FYI- These macros definition will be available in pal_config.h in PSA compliance test suite release.  Once you have pal_config.h, this json may not be required.

Reply:
This is the "mbed way" for configuration
We could add platform specific configuration in the future through this file